src/components/quote.js

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from 'react';
+import './quote.css';
+
+function Quote() {
+  const [quote, setQuote] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [errMsg, setErrMsg] = useState(null);
+
+  useEffect(() => {
+    let isMountd = true;
+    const API_KEY = 'RdZd0Xsx4bF/bK+ewY35lg==Gp1zeP14QV1BrS8W';
+
+    const fitchQuotes = async () => {
+      const res = await fetch(
+        'https://api.api-ninjas.com/v1/quotes?category=inspirational',
+        {
+          headers: { 'X-Api-Key': API_KEY },
+          contentType: 'application/json',
+        },
+      );
+
+      if (!res.ok) {
+        throw Error("Couldn't retrieve data");
+      }
+      return res.json();
+    };
+
+    fitchQuotes()
+      .then((data) => {
+        if (isMountd) {
+          setQuote(data[0].quote);
+          setLoading(false);
+          setErrMsg(null);
+        }
+      })
+      .catch((err) => {
+        if (isMountd) {
+          setQuote(null);
+          setLoading(false);
+          setErrMsg(err.message);
+        }
+      });
+    return () => {
+      isMountd = false;
+    };
+  }, []);
+
+  return (
+    <div className="quote">
+      {loading && <div>Loading...</div>}
+      {quote && <div>{quote}</div>}
```

Comment:
- [ **OPTIONAL**] You did an excellent job displaying the quote. However, I believe it would be beneficial to display the author's quote as well, so that visitors can learn who wrote the quote. :ok_hand: 🤓

Reply:
Thanks a lot for your advice 🤠